owner/secure_owner_api.go

```diff
@@ -140,6 +140,48 @@ func (owner *SecureOwnerAPI) InitSecureAPI(pubKey []byte) (string, error) {
 	return serverPubKey, nil
 }
 
+// Accounts Returns a list of accounts stored in the wallet
+// (i.e. mappings between user-specified labels and BIP32 derivation paths
+func (owner *SecureOwnerAPI) Accounts() (*[]libwallet.AccountPathMapping, error) {
+	params := struct {
+		Token string `json:"token"`
+	}{
+		Token: owner.token,
+	}
+	paramsBytes, err := json.Marshal(params)
+	if err != nil {
+		return nil, err
+	}
+
+	envl, err := owner.client.EncryptedRequest("accounts", paramsBytes, owner.sharedSecret)
+	if err != nil {
+		return nil, err
+	}
+
+	if envl == nil {
+		return nil, errors.New("Accounts: Empty RPC Response from grin-wallet")
```

Comment:
Being a bit picky but can you change it to the following so we keep it consistent?
```suggestion
		return nil, errors.New("OwnerAPI: Empty RPC Response from grin-wallet")
```